rpcserver.go

```diff
@@ -8060,6 +8060,7 @@ func (r *rpcServer) ForwardingHistory(ctx context.Context,
 			FeeMsat:     uint64(feeMsat),
 			AmtInMsat:   uint64(amtInMsat),
 			AmtOutMsat:  uint64(amtOutMsat),
+			HtlcIndex:   event.HtlcIndex,
```

Comment:
let's also update the test here to check the change,
https://github.com/lightningnetwork/lnd/blob/3707b1fb7032d98dc2a8e881d3ae83e61fbc35e3/itest/lnd_multi-hop-payments_test.go#L183-L184